Eat_and_Greet/src/comps/ChatCard/ChatCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import localstyle from './styles';
+
+interface ChatCardProps {
+  senderName: string;
+  message: string;
+  timestamp: string;
+  onPress?: () => void;
+}
+
+const ChatCard = ({ senderName, message, timestamp, onPress} : ChatCardProps) => {
```

Comment:
Wasn't on Figma, but I think we should pass in a prop based on whether or not the chat has been read yet. Maybe change the background color or add a small icon if unread?

Reply:
done